src/commands/aicommits.ts

```diff
@@ -76,6 +77,8 @@ export default async (
 	let message: string;
 	if (messages.length === 1) {
 		[message] = messages;
+		message = `${prefix} ${message}`;
```

Comment:
I understand the benefit of the space, but I'm afraid it becomes restrictive for those that don't want it and will later ask for a CLI flag to disable the space or something. But, I also haven't been able to come out with an example where they wouldn't want the space.

What do you think?

Reply:
In my case I like the space, because it is usually a ticker number like `FOO-123: ${message}`. However, if the space is not added, it could be added in the same prefix.

Reply:
I considered this and in the end I figured that for the few cases where no space would be needed people edit the commit after the fact like before. 

But on the other hand I don't personally mind removing the space. I use this with an alias that injects the punctuation I need so it would be trivial to include a space in that.

Reply:
@privatenumber Want me to get rid of the space?